In the locked_doesnt_build_without_cargo_lock() test, we will sometimes get
a slightly different error message content, this will catch both.
--- a/tests/common/errors.rs
+++ b/tests/common/errors.rs
@@ -77,7 +77,7 @@ pub fn locked_doesnt_build_without_cargo
             .source()
             .ok_or_else(|| format_err!("{}", err))?
             .to_string();
-        let error_msg = "`cargo metadata` exited with an error:     Updating crates.io index\nerror: the lock file";
+        let error_msg = "`cargo metadata` exited with an error:";
         if !err_string.starts_with(error_msg) {
             bail!("{:?}", err_string);
         }
